docroot/modules/custom/mass_content/src/Drush/Commands/MassContentCommands.php

```diff
@@ -425,6 +428,27 @@ public function searchAndReplaceMedia() {
                 }
               }
             }
+            elseif ($fieldType === 'link') {
+              foreach ($field as $key => $item) {
+                if ($item->uri) {
+                  if (strpos($item->uri, 'internal:') === 0) {
+                    $url = substr($item->uri, strlen('internal:'));
+                    $processed = $urlReplacementService->processLink($url);
+                    if ($processed['changed']) {
+                      $item->value = 'internal:' . $processed['link'];
```

Comment:
This is set to `$item->value` instead of `$item->uri` so that it works with `AutowireTrait`.